BEIMA.Client/src/pages/Authentication/LoginPage.js

```diff
@@ -0,0 +1,125 @@
+import { Card, Form, Button, Spinner } from "react-bootstrap";
+import { useOutletContext } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+
+import styles from './LoginPage.module.css'
+
+const LoginPage = () => {
+  const [setPageName] = useOutletContext();
+  const [username, setUsername] = useState("")
+  const [validUsername, setValidUsername] = useState(false)
+  const [password, setPassword] = useState("")
+  const [validPassword, setValidPassword] = useState(false)
+  const [remember, setRemember] = useState(false)
+  const [touched, setTouched] = useState(false)
+  const [submitting, setSubmitting] = useState(false)
+
+  useEffect(() => {
+    setPageName('Login')
+  }, [setPageName])
+
+
+  const ValidateUsername = (username) => {
+    let valid = true
+    if(username.length < 8 || username.length > 24){
+      valid = false
+    }
+    return valid
+  }
+
+  const ValidatePassword = (password) => {
+    let valid = true
+    if(password.length < 8 || password.length > 24){
```

Comment:
I don't think we should have a cap on password length, or I think we should make it ridiculously big like 100 characters or something.